crates/client/src/api.rs

```diff
@@ -0,0 +1,22 @@
+use anyhow::Result;
+use reqwest::Url;
+use serde::Deserialize;
+use serde_json::Value;
+
+#[derive(Deserialize, Debug)]
+pub struct Stars {
+    pub amount: Option<u64>,
+}
+
+impl Stars {
+    pub async fn get_amount() -> Result<Self> {
+        let url = format!("https://api.github.com/repos/rustacean-sh/rustacean.sh");
```

Comment:
Given that we are not interpolating strings we can use a string slice directly.

```suggestion
        let url = "https://api.github.com/repos/rustacean-sh/rustacean.sh";
```